COOKBOOK.md

```diff
@@ -160,6 +160,32 @@ index 5bc1fdc..54fe19f 100644
  };
 ```
 
+## Using a template
+This command takes advantage of Ailly's templating feature.
+
+In `path/to/javascript-prompt.yaml` there is a `code.tests` property with information common to a project's tests:
+```yaml
+...
+code:
+  tests: |
+      <tests>
```

Comment:
`</tests>`?

Reply:
It's truncated because the real one is huge, but I'll clean it up.